Unreachable URL in `RequireExplicitNullMarking` error message

> Hi @msridhar, I noticed one more cosmetic thing today when addressing `RequireExplicitNullMarking` warnings. I'm commenting here as I'm unsure if this really deserves opening a new issue.
> 
> The second line of the error hints that there is a https://errorprone.info/bugpattern/RequireExplicitNullMarking URL:
> 
> ```
> [ERROR] /Users/stefano/Projects/spring-batch/spring-batch-core/src/main/java/org/springframework/batch/core/annotation/AfterChunk.java:[37,9] [RequireExplicitNullMarking] [RequireExplicitNullMarking] Top-level classes must either be directly annotated with @NullMarked/@NullUnmarked, be in a package that is explicitly @NullMarked/@NullUnmarked, or be in a module that is explicitly @NullMarked.
>     (see https://errorprone.info/bugpattern/RequireExplicitNullMarking)
> ```
> 
> but that URL returns a 404.
> 
> Is that intended? 

 _Originally posted by @scordio in [#1364](https://github.com/uber/NullAway/issues/1364#issuecomment-3617031199)_

--- a/nullaway/src/main/java/com/uber/nullaway/RequireExplicitNullMarking.java
+++ b/nullaway/src/main/java/com/uber/nullaway/RequireExplicitNullMarking.java
@@ -76,7 +76,6 @@ public final class RequireExplicitNullMarking extends BugChecker
 
   @Override
   public String linkUrl() {
-    // trailing space is deliberate to help ensure clickability from terminals
-    return "https://github.com/uber/NullAway/wiki/JSpecify-Support#requireexplicitnullmarking-checker ";
+    return "https://github.com/uber/NullAway/wiki/JSpecify-Support#requireexplicitnullmarking-checker";
   }
 }

--- a/nullaway/src/test/java/com/uber/nullaway/RequireExplicitNullMarkingTest.java
+++ b/nullaway/src/test/java/com/uber/nullaway/RequireExplicitNullMarkingTest.java
@@ -41,7 +41,7 @@ public class RequireExplicitNullMarkingTest {
         .addSourceLines(
             "test/MissingAnnotation.java",
             "package test;",
-            "// BUG: Diagnostic contains: https://github.com/uber/NullAway/wiki/JSpecify-Support#requireexplicitnullmarking-checker ",
+            "// BUG: Diagnostic contains: https://github.com/uber/NullAway/wiki/JSpecify-Support#requireexplicitnullmarking-checker",
             "class MissingAnnotation {",
             "}")
         .doTest();

